torch/ao/quantization/pt2e/utils.py

```diff
@@ -47,6 +47,9 @@ def fold_bn_weights_into_conv_node(
     conv_w = _get_tensor_constant_from_node(conv_weight_node, m)
     conv_b = _get_tensor_constant_from_node(conv_bias_node, m)
     transpose = not (conv_node.target == torch.ops.aten.conv2d.default)
+    if conv_node.target == torch.ops.aten.convolution.default:
+        assert type(conv_node.args[6]) is bool
+        transpose = conv_node.args[6]
```

Comment:
could you add a TODO to remove these

Reply:
Thanks for the comments, TODO added here and others in this diff.